### PR TITLE
[PMIG-500] Fix Snyk vulnerabilities

### PR DESCRIPTION
**Pull request by lfernandes-ps**
### Summary:
This PR updates `color-thief`'s direct and indirect dependencies to versions that aren't susceptible to known vulnerabilities.

**TL;DR:** Fixed all vulnerabilities except 1 medium severity issue which is not fixable at this time without a considerable amount of work on third-party packages. Regardless, the Snyk Security check still passes ✅ 

**More details:**
⚠️ There's only one [medium severity issue (SSRF)](https://security.snyk.io/vuln/SNYK-JS-REQUEST-3361831) that's left, which has to do with the `request` package – a dependency of `get-pixels` which is a direct dependency from `color-thief`. However, there's no proposed fix, upgrade or patch, given the [`request` package has been deprecated](https://github.com/request/request/issues/3142) in 2019. Until now no direct fix has been created, since it requires changing the `get-pixels` package and removing the `request` dependency.
Recently, a [PR was created](https://github.com/scijs/get-pixels/pull/65) for `get-pixels` that replaces `request` with `node-fetch`, but it's two weeks old and it hasn't seen any traction yet, so I don't have high hopes that it will be merged any time soon.

> More info about SSRF vulnerability: https://github.com/request/request/issues/3442

### How this was tested:
Firstly, after updating the packages, I ran the `color-thief` project and verified it compiled and loaded properly in the browser. I also ran unit and browser tests, which passed ✅

The only project that uses/depends on our version of `color-thief` is `picmonkey/www` (a.k.a. SEO). To ensure it would work with the updated packages in `color-thief`, I initially cleaned the SEO project of all previously built assets, cache and node_modules, and installed and built the project from scratch – this was done to remove as many variables as possible and create a controlled environment.
After verifying everything was working fine, I then edited the `package.json` file of `/www` and changed the `color-thief` dependency so it would use my local version (with the updated packages). I then cleaned the project, built it from scratch and ran it once again to find everything was working fine.
I also ran SEO's tests, which passed ✅

### CODE REVIEW

**Review by michael-fischl — APPROVED** [review]
Looks good to me!

However did you also look at the forks of [get-pixel](https://github.com/scijs/get-pixels/forks?include=active&page=1&period=2y&sort_by=last_updated)? Seems like people are updating dependencies with little other changes so the overhead would be small. We could _trust_ one of them and use that one or create our own fork with those updated dependencies. But I think thats more of a follow-up matter since the current change you have is clean enough and makes sense. Just food for thought.

**Review by ps-mzamette — APPROVED** [review]

**Review by ps-mveith — APPROVED** [review]
